changelog.md

```diff
@@ -58,6 +58,7 @@ Following properties to be removed from output and/or ignored on input:
 | [Currency value](operations/finance.md#currency-value) | `Tax` | --- |
 | [Currency value](operations/finance.md#currency-value) | `TaxRate` | --- |
 | Whole `UnitCost` object | --- | [Amount](operations/finance.md#amount-value) |
+| [Bill options](operations/finance.md#bill-options) | `Rebated` | --- |
```

Comment:
@KaliCZ @abdallahbeshi this is not enough. We need to specify how the new logic works and what can be used instead of this bool getter. This has to be described here.

Reply:
I mean, help them how to change it. We need to say them, how they can stop using this property but get the same information.

Reply:
Makes sense. I will extend `accountingItems/getAll` request with RebatedItemIds then update the documentation.